docs/facilitation/Remote.md

```diff
@@ -34,17 +36,20 @@ In a remote configuration, the introduction message is really important as it is
   - You can use the main session chat to ask for help
   - We will be posting notifications through the main session channel, *so keep an eye on it*, you'll get notifications about timings, troubleshooting, additional instructions...
   - [Miro]
-    - We'll use Miro embedded timers
+    - We'll use Miro embedded timers to remind you to jump to the next step
     - We'll be using the 'bring everyone to me' to get you back all together at the end
-    - We tried our best to make the board self-explanatory, but the markdown files in the repo go in more details if you are stuck
+    - We tried our best to make the board self-explanatory, just follow the boxes and the flow
+    - In case you are lost and need more details, the markdown files in the repo go in more details if you are stuck
```

Comment:
"In case you are lost ... if you are stuck": this feels a bit too much, don't you think?